### DISCUSSION TITLE
What happens to the "custom settings" if the Content Types to which they are added get deleted?

### DISCUSSION THREAD

**Discussion by MarGraz** [OrchardCMS/OrchardCore#15643] [Q&A]
Following the discussion in [this topic](https://github.com/OrchardCMS/OrchardCore/discussions/15612) about adding "custom settings" to an Orchard Core `BagPart, I have opened this new topic as suggested [here](https://github.com/OrchardCMS/OrchardCore/discussions/15612#discussioncomment-8952471). I would like to ask: 

1. What happens to these custom settings if the Content Types **to which they are added get deleted**? 
2. Is there a way to establish a relationship with these custom settings so that Orchard Core **can automatically remove them** to maintain a clean database?

Thank you

**Comment by hishamco** [comment]
1. The custom settings should remind AFAIK because it's a content type with a `CustomSettings` stereotype
2. In terms of relationships there: Lists & BagPart

For more information, please read the following:

https://docs.orchardcore.net/en/latest/docs/topics/content-relationships/
https://github.com/OrchardCMS/OrchardCore/issues/4494
